crates/bitwarden/src/lib.rs

```diff
@@ -59,6 +59,7 @@ pub mod platform;
 #[cfg(feature = "secrets")]
 pub mod secrets_manager;
 mod util;
+pub mod vault;
```

Comment:
I think we can move the mobile check here?

Reply:
Oh yeah, that's true, changed in 1ddda6e